### DISCUSSION TITLE
Support for 4 inch Smart panlee ESP32-S3 ZX3D95CE01S-TR-4848 ?

### DISCUSSION THREAD

**Discussion by floco** [toverainc/willow#269] [General]
Hello everyone,

This looks to be another ESP32-S3 with screen and mic: https://www.aliexpress.com/i/1005005235784548.html
With bigger screen. Made by smart panlee not espressif.
Datasheet:
https://img01.71360.com/file/read/www2/M00/44/23/rBwBEmRaCvuACmyBAB9xVQ-_bcE696.pdf?dl=1&dlf=ZX3D95CE01S-AR-4848+Datasheet.pdf
Any idea if that would be supported now or in the future ?

**Comment by kristiankielhofner** [comment]
Hi!

I've looked over the datasheet and support is possible. However, there are a few issues:

1) Their reference code is based on IDF 5.1. We just added a branch for it yesterday but it is still very early.

2) Our current focus is audio first with minimal GUI. We use all of the same components but driving the display and making full use of it other than our simple UI is a significant amount of work. Our current devices all have a much smaller and lower resolution display and we target practical GUIs for them.

3) Our intention is to support widely available devices with committed long term manufacturing support. The concern with a lot of these devices available on Aliexpress is they tend to come and go - here one day and not the next. If we add hardware support we want to make sure it's hardware people can actually get. Already this device only shows 19 in stock and it's a complete unknown if there will be more stock and for how long. Even their own IDF code is a single initial commit from two months ago - which doesn't point to very involved support from the manufacturer.

**Reply by floco** [reply]
Hello Kristian,

Thanks for your feedback. 
Your points makes total sense and especially point 3 is indeed a concern.
I will go with the ESP32-S3-BOX then, thanks.

**Reply by kristiankielhofner** [reply]
Ironically the ESP-BOX is no longer manufactured (but we will continue to support it indefinitely). It was originally intended to be a very low-volume development platform. After the release of Willow and surge in demand Espressif made the [ESP-BOX-3](https://www.espressif.com/en/news/ESP32-S3-BOX-3) which offers a lot of improvements. We support the BOX-3 today but their early release is currently sold out. We have been informed by Espressif that the full manufacturing run (with proper injection molded enclosure!) will be available with ample stock in mid-October.
